### DISCUSSION TITLE
PDM microphone

### DISCUSSION THREAD

**Discussion by JPSvdW** [zephyrproject-rtos/zephyr#31441] [Q&A]
Hi, has anyone successfully connected a PDM microphone using the PDM driver?  There seems to be no PDM microphone sample that can guide me.

**Comment by henrikbrixandersen** [comment]
Zephyr does have an API for digital microphones (see https://github.com/zephyrproject-rtos/zephyr/blob/master/include/audio/dmic.h). Which board/SoC are you using?

**Reply by HiberXen** [reply]
nrf52832

**Comment by henrikbrixandersen** [comment]
Please see https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/drivers/audio/dmic
